lib/file-operations.js

```diff
@@ -149,9 +200,24 @@ function updateMetadata(fd, file, callback) {
           file.stat.atime = timesDiff.atime;
           file.stat.mtime = timesDiff.mtime;
         }
+        if (ownerDiff) {
+          owner(fchmodErr, futimesErr);
```

Comment:
missing the `return` here so the `callback` below would always be called at the wrong time